src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -62,143 +45,43 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
-                type: NotificationType.SUCCESS,
-            });
-        } catch (err) {
-            const errMessage = err.message ? `, (${err.message})` : '';
-            NotificationStore.addNotification({
-                message: `Tallennus epäonnistui${errMessage}`,
-                type: NotificationType.ERROR,
-            });
-        }
-        this.setState({ isLoading: false });
-    }
-
-    private markInvalidFields = (field: string, isValid: boolean) => {
-        this.setState({
-            invalidFieldsMap: {
-                ...this.state.invalidFieldsMap,
-                [field]: isValid,
-            },
-        });
-    }
-
-    private isFormValid = () => {
-        return !Object.values(this.state.invalidFieldsMap)
-            .some(fieldIsValid => !fieldIsValid);
-    }
-
-    public onChange = (property: string, value: any, validationResult?: IValidationResult) => {
+    public selectTab = (selectedTab: string) => () => {
         this.setState({
-            routePath: { ...this.state.routePath!, [property]: value },
-            hasModifications: true,
+            selectedTab,
         });
-        if (validationResult) {
-            this.markInvalidFields(property, validationResult!.isValid);
-        }
-    }
-
-    private redirectToNewRoutePathView = () => {
-        const routePath = this.state.routePath;
-        if (!routePath) return;
-
-        const newRoutePathLink = routeBuilder
-        .to(subSites.newRoutePath, { routeId: routePath.routeId, lineId: routePath.lineId })
-        .toLink();
-
-        this.props.routePathStore!.setRoutePath(this.state.routePath);
-        navigator.goTo(newRoutePathLink);
-
     }
 
     public render(): any {
-        // tslint:disable-next-line:max-line-length
-        const message = 'Reitin suunnalla tallentamattomia muutoksia. Oletko varma, että poistua näkymästä? Tallentamattomat muutokset kumotaan.';
-
         if (this.state.isLoading) {
             return (
                 <div className={s.routePathView}>
                     <Loader size={Loader.MEDIUM}/>
                 </div>
             );
         }
-        if (!this.state.routePath) return 'Error';
+        if (!this.state.routePath) return;
         return (
-        <div className={classnames(s.routePathView, s.form)}>
-            <div className={s.formSection}>
-                <ViewHeader
-                    header={`Reitin suunta`}
-                    closePromptMessage={this.state.hasModifications ? message : undefined}
-                >
-                    <Button
-                        onClick={this.toggleEditing}
-                        type={ButtonType.SQUARE}
-                        text={this.state.isEditingDisabled ? 'Muokkaa' : 'Peruuta'}
+            <div className={s.routePathView}>
+                <div className={s.flexInnerRow}>
+                    <RoutePathViewTabButtons
+                        tabs={['Reitinsuunta', 'Solmut ja linjat']}
```

Comment:
What about 'Reitinsuunnan tiedot'?

Reply:
For me "Reitinsuunta" is fine, its in sync with "Solmut ja linjat"